--------------------------------------------------------------------------------
Step: 1, Block: 2768971, Master: 0 Hash: 231cba3829bee58f27a930bbd4c475fdadb5ba486672cd2fff76890b9f806735
Failed to sync master state using deltas.
Downloading model from model-5F4UUMWF41GsLFvwVpigmoxTKVngDP6C7utvECgtQB83U3fJ-1726001155.pt@decis
/home/ubuntu/frontend/venv/lib/python3.10/site-packages/torch/cuda/__init__.py:654: UserWarning: Can't initialize NVML
  warnings.warn("Can't initialize NVML")
Traceback (most recent call last):
  File "/home/ubuntu/frontend/bistro/miner.py", line 122, in main
    master = download_model( metadata = master_meta, device = config.device, CLIENT = CLIENT )
  File "/home/ubuntu/frontend/bistro/common.py", line 349, in download_model
    new_model_state_dict = torch.load(unique_temp_file, map_location=torch.device(device), weights_only=True)  # Load the model state dict
  File "/home/ubuntu/frontend/venv/lib/python3.10/site-packages/torch/serialization.py", line 1090, in load
    return _load(opened_zipfile,
  File "/home/ubuntu/frontend/venv/lib/python3.10/site-packages/torch/serialization.py", line 1525, in _load
    result = unpickler.load()
  File "/home/ubuntu/frontend/venv/lib/python3.10/site-packages/torch/_weights_only_unpickler.py", line 342, in load
    self.append(self.persistent_load(pid))
  File "/home/ubuntu/frontend/venv/lib/python3.10/site-packages/torch/serialization.py", line 1492, in persistent_load
    typed_storage = load_tensor(dtype, nbytes, key, _maybe_decode_ascii(location))
  File "/home/ubuntu/frontend/venv/lib/python3.10/site-packages/torch/serialization.py", line 1466, in load_tensor
    wrap_storage=restore_location(storage, location),
  File "/home/ubuntu/frontend/venv/lib/python3.10/site-packages/torch/serialization.py", line 1392, in restore_location
    return default_restore_location(storage, str(map_location))
  File "/home/ubuntu/frontend/venv/lib/python3.10/site-packages/torch/serialization.py", line 414, in default_restore_location
    result = fn(storage, location)
  File "/home/ubuntu/frontend/venv/lib/python3.10/site-packages/torch/serialization.py", line 392, in _deserialize
    return obj.to(device=device)
  File "/home/ubuntu/frontend/venv/lib/python3.10/site-packages/torch/storage.py", line 187, in to
    return _to(self, device, non_blocking)
  File "/home/ubuntu/frontend/venv/lib/python3.10/site-packages/torch/_utils.py", line 75, in _to
    with device_module.device(device):
  File "/home/ubuntu/frontend/venv/lib/python3.10/site-packages/torch/cuda/__init__.py", line 386, in __enter__
    self.prev_idx = torch.cuda._exchange_device(self.idx)
  File "/home/ubuntu/frontend/venv/lib/python3.10/site-packages/torch/cuda/__init__.py", line 327, in _lazy_init
    queued_call()
  File "/home/ubuntu/frontend/venv/lib/python3.10/site-packages/torch/cuda/__init__.py", line 195, in _check_capability
    capability = get_device_capability(d)
  File "/home/ubuntu/frontend/venv/lib/python3.10/site-packages/torch/cuda/__init__.py", line 451, in get_device_capability
    prop = get_device_properties(device)
  File "/home/ubuntu/frontend/venv/lib/python3.10/site-packages/torch/cuda/__init__.py", line 467, in get_device_properties
    if device < 0 or device >= device_count():
  File "/home/ubuntu/frontend/venv/lib/python3.10/site-packages/torch/cuda/__init__.py", line 842, in device_count
    nvml_count = _device_count_amdsmi() if torch.version.hip else _device_count_nvml()
  File "/home/ubuntu/frontend/venv/lib/python3.10/site-packages/torch/cuda/__init__.py", line 798, in _device_count_nvml
    raw_cnt = _raw_device_count_nvml()
  File "/home/ubuntu/frontend/venv/lib/python3.10/site-packages/torch/cuda/__init__.py", line 652, in _raw_device_count_nvml
    rc = nvml_h.nvmlInit()
KeyboardInterrupt
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/home/ubuntu/frontend/bistro/miner.py", line 241, in <module>
    main( config )
  File "/home/ubuntu/frontend/bistro/miner.py", line 213, in main
    handle_exit_signal( None, None, history, run )
  File "/home/ubuntu/frontend/bistro/miner.py", line 64, in handle_exit_signal
    run.delete()
